src/components/pages/ParentContact/ParentContactContainer.js

```diff
@@ -0,0 +1,122 @@
+import React, { useState, useEffect } from 'react';
+import { Layout, Row, Col } from 'antd';
+import img2 from './img2.png';
+import ParentNavTopBar from '../../common/ParentNavTopBar';
+
+const ParentContactContainer = () => {
+  const [name, setName] = useState('');
+  const [email, setEmail] = useState('');
+  const [message, setMessage] = useState('');
+  const [item, setItem] = useState([]);
+
+  const handleSubmit = e => {
+    e.preventDefault();
+    setItem([...item, { name: name, email: email, message: message }]);
+    setName('');
+    setEmail('');
+    setMessage('');
+  };
+
+  useEffect(() => {
+    const data = localStorage.getItem('data');
+    if (data) {
+      setItem(JSON.parse(data));
+    }
+  }, []);
+
+  useEffect(() => {
+    localStorage.setItem('data', JSON.stringify(item));
+  });
+  return (
+    <>
```

Comment:
Looks like you aren't following the file structure of the rest of the application - I would move all of this into a renderParentContact file and then import it into the ParentContact container. This way of settings things up is called the redux container pattern - you can learn about it here https://www.thegreatcodeadventure.com/the-react-plus-redux-container-pattern/